test/e2e/shared/suite.go

```diff
@@ -169,6 +169,9 @@ func Node1BeforeSuite(e2eCtx *E2EContext) []byte {
 
 	CreateAWSClusterControllerIdentity(e2eCtx.Environment.BootstrapClusterProxy.GetClient())
 
+	By("Setting up AWS static credentials")
+	SetupStaticCredentials(e2eCtx)
```

Comment:
Do we need to create static credentials for unmanaged tests?

Reply:
This should do it for both the managed and unmanaged

Reply:
We are using AWSClusterControllerIdentity in unmanaged clusters, should not need this.

Reply:
Sorry @sedefsavas - i misread that

Reply:
This doesn't affect any test unless the test uses AWSClusterStaticIdentity. We can set it up for only managed test suite but someone may want to use AWSClusterStaticIdentity for unmanaged suite in the future.

Reply:
Just saw new comments :) Either way is fine. Let me know what you think.

Reply:
We can add appropriate identity creation to specific tests like we do in nested multi-tenancy tests.
I'd opt for not creating unused objects that may or may not be used in the future.

Reply:
Ok, so moving this back into the managed suite then, where it can be called from both tests

Reply:
Not moving back to the managed suite but added conditions for the setup and deletion
```
	if e2eCtx.IsManaged {
		By("Setting up AWS static credentials")
		SetupStaticCredentials(e2eCtx)
	}
```

Reply:
I think we may want to split suite.go for managed and unmanaged in the long term as the needs are different.